### PR TITLE
Updating Key Vault Java SDK to use the latest Runtime Client.

### PR DESCRIPTION
**Pull request by pomortaz**

### CODE REVIEW

**Comment by jianghaolu** [comment]
beta5 / beta6 mismatch. Will fix in a separate PR after master is fixed.

**Comment by pomortaz** [comment]
@jianghaolu I don't understand? So the master has upgraded to client runtime version 1.0.0-beta6 and I need to update my code to use beta6? Why it is not compatible? 
What do you mean by "Will fix in a separate PR after master is fixed"? Is it actionable by me?

**Comment by jianghaolu** [comment]
@pomortaz You are all set. It's an action we need to take!

**Comment by jianghaolu** [comment]
@pomortaz  Sorry for the confusion!

File: azure/pom.xml [hunk 1]
```diff
@@ -110,6 +110,31 @@
       <version>1.0.0-beta5-SNAPSHOT</version>
     </dependency>
     <dependency>
+      <groupId>com.microsoft.azure</groupId>
```
Comment (jianghaolu) [inline azure/pom.xml]: This is not correct. `azure` is the roll-up management client and shouldn't depend on data plane packages. I'll remove in a separate PR.

**Pull request merged**